resource-timing/resources/all_resource_types.htm

```diff
@@ -6,6 +6,35 @@
         <link rel="shortcut icon" href="empty.py?favicon">
     </head>
     <body background='blue.png?id=body'>
+        <script>
+            let count = 0;
+            let totalEventSourceCount = 0;
+
+            function perf_observer(list, observer) {
+                const entries = list.getEntries();
+                const eventSourceCount = entries.filter(e => e.name.includes('eventsource')).length;
+
+                if(eventSource > 0) {
+                    totalEventSourceCount += eventSourceCount;
+                } 
+                
+                // exclude eventSource since it arrives repeatedly
+                count += entries.filter(e => !e.name.includes('eventsource')).length;
+                
+                if(count === 34 && totalEventSourceCount > 0) {
+                    // we are done only if all static resources arrived and at least one event source
+                    observer.disconnect();
+                    parent.on_all_resources_loaded();
+                }
+            } 
+        
+            const observer = new PerformanceObserver(perf_observer); 
+            observer.observe({entryTypes: ["resource"]});
```

Comment:
There is a resource for the body background above, is that supposed to be included here or no? Seems like it could trigger some race condition. I think we should set the value below if it is meant to be included, remove the background if it's not.

Reply:
per offline discussion - moved higher to head element to reduce the race.